content/en/docs/reference/glossary/node.md

```diff
@@ -14,5 +14,4 @@ tags:
 
 <!--more--> 
 
-A worker machine may be a VM or physical machine, depending on the cluster. It has the {{< glossary_tooltip text="Services" term_id="service" >}} necessary to run {{< glossary_tooltip text="Pods" term_id="pod" >}} and is managed by the master components. The {{< glossary_tooltip text="Services" term_id="service" >}} on a node include Docker, kubelet and kube-proxy.
-
+A worker machine may be a VM or physical machine, depending on the cluster. It has local daemons or services necessary to run {{< glossary_tooltip text="Pods" term_id="pod" >}} and is managed by the master components. The daemons (or operating system services) on a node include {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}, kube-proxy, and a container runtime such as {{< glossary_tooltip term_id="docker" >}}.
```

Comment:
Possible edits:
- could this paragraph start with, `A non-master node may be ... , with local daemons or services necessary to run Pods. Each non-master node is managed by the Kubernetes master components. See, https://kubernetes.io/docs/concepts/
-  do you need `text=Docker` for the `docker` glossary term?

Reply:
Kubernetes [self-hosting](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#self-hosting) is in alpha. With that in mind, is the concept of a “non-master node” something I should emphasize?
A lot of the documentation assumes that Node is equivalent to “worker node”, and the “control plane node” concept seems deprecated. I'm also considering https://github.com/kubernetes/kubeadm/issues/1451 in respect of this.


> do you need text=Docker for `docker` glossary term?

I don't think I do; I'm happy to use whatever name the glossary term with id `docker` appears as in the glossary

Reply:
- not sure which concept name is acceptable these days: non-master vs. worker node
- the Docker glossary text is fine
- Just re-read. Text seems okay. I'm not sure `(or operating system services)` is needed, but your call.

Reply:
I'm wary of writing daemon because there _are_ people who run Kubernetes with just Windows OS nodes (and a managed control plane).

Maybe I'm overthinking it?

Reply:
I don't think you have to discriminate or get too detailed here -- it is a glossary definition. Keeping it simple seems to be the best approach 😄

Reply:
Revised. How's the current iteration looking?

Reply:
Looks very close to the current version on kubernetes.io.
Tweaks seems fine. 👍